app/models/event.rb

```diff
@@ -0,0 +1,20 @@
+class Event < ActiveRecord::Base
+  belongs_to :user
+
+  enum event_type: {
+    account_created: 1,
+    phone_confirmed: 2,
+    password_changed: 3,
+    phone_changed: 4,
+    email_changed: 5,
+    authenticator_enabled: 6,
+    authenticator_disabled: 7
+  }
+
+  validates :event_type, presence: true
+
+  def event_type_to_s
+    raise 'Invalid event type' unless Event::event_types.keys.include?(event_type)
+    I18n.t('event_types.' + event_type.to_s)
+  end
```

Comment:
How about moving this to an EventDecorator class since this is only used by a view. This will also allow you to call the decorator method `event_type`, which reads better IMO. Having to add `to_s` to the end of a method name smells strange to me.